### PR TITLE
Hotfix/user serializer expansion

### PR DESCRIPTION
**Pull request by MaxWasUnavailable**
## Description

> Please include a summary of the change and which issue is resolved. List any dependencies that are required for this change.

Created NestedUserExtraInfoSerializer, which is added as a nested serializer to the User serializer; adding its info the User object when requested through the User serializer.

We now also create the linked UserExtraInfo object automatically when a user registers via the RegisterSerializer or is created through the regular UserSerializer.

## Type of change

> Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

- Created new users through API & Register routes.
- Checked UserExtraInfo in admin view.

## Checklist:

> Please check [x] all that apply.

- [x] My code follows similar styling to existing code in this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, at the very least in hard-to-understand areas
- [ ] I have made corresponding changes to documentation / READMEs
- [x] My changes generate no new warnings
- [x] I have remade my frontend and backend Docker containers to ensure my changes do not break the setup for these
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: backend/account/serializers.py [hunk 1]
```diff
@@ -5,10 +5,42 @@
 from .models import UserExtraInfo
 
 
+class NestedUserExtraInfoSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = UserExtraInfo
+        fields = ['in_game_name', 'discord_name', 'forum_name', 'bio', 'home_origin', 'profile_picture', 'profile_picture']
```
Comment (pascalstockert) [inline backend/account/serializers.py]: 2x profile_picture?

Comment (pascalstockert) [reply backend/account/serializers.py]: Actually, can't these be removed entirely as the picture is saved outside of the extra user info?

Comment (MaxWasUnavailable) [reply backend/account/serializers.py]: Profile picture is stored in the UserExtraInfo model currently. Considering you asked me to add the profile picture outside of that, I tossed it in there as well. The UserExtraInfo info is added through the UserExtraInfo serializer that's nested in the User serializer.

Comment (MaxWasUnavailable) [reply backend/account/serializers.py]: Oh, whoops, let me fix that.

**Review by pascalstockert — APPROVED** [review]

**Comment by MaxWasUnavailable** [comment]
@pascalstockert Should be fixed 👌🏻

**Pull request merged**